cakechat/api/response.py

```diff
@@ -1,7 +1,8 @@
 import random
+from six.moves import xrange
 
-from cakechat.api.config import PREDICTION_MODE, NUM_BEST_CANDIDATES_TO_PICK_FROM, SAMPLING_ATTEMPTS_NUM, \
-    DEFAULT_RESPONSE
+from cakechat.api.config import (PREDICTION_MODE, NUM_BEST_CANDIDATES_TO_PICK_FROM, SAMPLING_ATTEMPTS_NUM,
```

Comment:
Parentheses in multiline imports are not required by python 3. It handles both styles. We prefer '\' instead parentheses. Please fix it back

Reply:
[PEP8 advises the avoidance of backslashes](https://www.python.org/dev/peps/pep-0008/#id19) because one whitespace character to the right of the backslash breaks the script but is invisible to the reader.

Reply:
You are right, didn't know about this pep8 advise. But anyway, this PR is about supporting both python2 and python3, not about pep8 fixes. If you'd like, you can create new PR with this style fix.